src/SparkpostServiceProvider.php

```diff
@@ -40,19 +40,20 @@ public function register()
 
             $manager->extend('sparkpost', function() {
 
-                $config = $this->app['config']->get('services.sparkpost', []);
-                $options = isset($config['guzzle']) ? $config['guzzle'] : [];
-                $client = new Client($options);
+                $config             = $this->app['config']->get('services.sparkpost', []);
+                $sparkpost_options  = isset($config['options']) ? $config['options'] : false;
```

Comment:
Please can you change default to `[]`
and switch to camelCase variable names: `$sparkpostOptions, `$guzzleOptions`